From 35f8908667750d2201e84a204264e2db988d3ae4 Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Sun, 25 Dec 2011 22:57:11 +0100
Subject: [PATCH] Add IDs to create HTML anchors for (almost) each version.

When the minor patch level was not indicated, it was set to zero.  When more
than one minor patch level was indicated, the lowest was used.

See: http://bugs.debian.org/639663
---
 upgrading-checklist.sgml |   88 +++++++++++++++++++++++-----------------------
 1 files changed, 44 insertions(+), 44 deletions(-)

diff --git a/upgrading-checklist.sgml b/upgrading-checklist.sgml
index 80a4810..41220a0 100644
--- a/upgrading-checklist.sgml
+++ b/upgrading-checklist.sgml
@@ -34,7 +34,7 @@ picking your way through this list.
 
 <chapt> The checklist
 
-<sect> Version 3.9.3.0
+<sect id="3.9.3.0"> Version 3.9.3.0
 <p>
 
 Unreleased.
@@ -72,7 +72,7 @@ Unreleased.
   </item>
 </taglist></p>
 
-<sect> Version 3.9.2.0
+<sect id="3.9.2.0"> Version 3.9.2.0
 <p>
 
 Released April, 2011.
@@ -142,7 +142,7 @@ Released April, 2011.
   </item>
 </taglist></p>
 
-<sect> Version 3.9.1.0
+<sect id="3.9.1.0"> Version 3.9.1.0
 <p>
 
 Released July, 2010.
@@ -239,7 +239,7 @@ Released July, 2010.
   </item>
 </taglist></p>
 
-<sect> Version 3.9.0.0
+<sect id="3.9.0.0"> Version 3.9.0.0
 <p>
 
 Released June, 2010.
@@ -338,7 +338,7 @@ Released June, 2010.
   </item>
 </taglist></p>
 
-<sect> Version 3.8.4.0
+<sect id="3.8.4.0"> Version 3.8.4.0
 <p>
 
 Release Jan 2010.
@@ -359,7 +359,7 @@ Release Jan 2010.
   </item>
 </taglist></p>
 
-<sect> Version 3.8.3.0
+<sect id="3.8.3.0"> Version 3.8.3.0
 <p>
 Released Aug 2009.
 
@@ -415,7 +415,7 @@ Released Aug 2009.
   </item>
 </taglist></p>
 
-<sect> Version 3.8.2.0
+<sect id="3.8.2.0"> Version 3.8.2.0
 <p>
 
 Released Jun 2009.
@@ -455,7 +455,7 @@ Released Jun 2009.
   </item>
 </taglist></p>
 
-<sect> Version 3.8.1.0
+<sect id="3.8.1.0"> Version 3.8.1.0
 <p>
 
 Released  Mar 2009.
@@ -513,7 +513,7 @@ Released  Mar 2009.
   </item>
 </taglist></p>
 
-<sect> Version 3.8.0.0
+<sect id="3.8.0.0"> Version 3.8.0.0
 <p>
 
 Released  Jun 2008.
@@ -592,7 +592,7 @@ Released  Jun 2008.
 <item>Underscore (_) is allowed in debconf template names.     
 </taglist></p>
 
-<sect> Version 3.7.3.0
+<sect id="3.7.3.0"> Version 3.7.3.0
 <p>
 
 Released  Dec 2007.
@@ -634,7 +634,7 @@ Released  Dec 2007.
   be referenced rather than quoted in debian/copyright.
 </taglist></p>
 
-<sect> Version 3.7.2.2
+<sect id="3.7.2.2"> Version 3.7.2.2
 <p>
 
 Released  Oct 2006.
@@ -644,7 +644,7 @@ Released  Oct 2006.
     should to a must)</item>   
 </taglist></p>
 
-<sect> Version 3.7.2.0
+<sect id="3.7.2.0"> Version 3.7.2.0
 <p>
 
 Released  Apr 2006.
@@ -653,7 +653,7 @@ Released  Apr 2006.
 <tag>11.5</tag> <item>Revert the cgi-lib change. </item>
 </taglist></p>
 
-<sect> Version 3.7.1.0
+<sect id="3.7.1.0"> Version 3.7.1.0
 <p>
 
 Released  Apr 2006.
@@ -673,7 +673,7 @@ Released  Apr 2006.
   they do so, they must pre-depend on <tt>x11-common (&gt;= 1:7.0.0)</tt>
 </taglist></p>
 
-<sect> Version 3.7.0.0
+<sect id="3.7.0.0"> Version 3.7.0.0
 <p>
 
 Released  Apr 2006.
@@ -714,7 +714,7 @@ Released  Apr 2006.
   is gone.
 </taglist></p>
 
-<sect> Version 3.6.2.0
+<sect id="3.6.2.0"> Version 3.6.2.0
 <p>
 
 Released  2005
@@ -734,7 +734,7 @@ Released  2005
 </item>
 </taglist></p>
 
-<sect> Version 3.6.1.0
+<sect id="3.6.1.0"> Version 3.6.1.0
 <p>
 
 Released  Aug 2003.
@@ -745,7 +745,7 @@ Released  Aug 2003.
   user prompts are now deprecated.
 </taglist></p>
 
-<sect> Version 3.6.0
+<sect id="3.6.0.0"> Version 3.6.0
 <p>
 
 Released  Jul 2003.
@@ -789,7 +789,7 @@ Released  Jul 2003.
   target.
 </taglist></p>
 
-<sect> Version 3.5.10
+<sect id="3.5.10.0"> Version 3.5.10
 <p>
 
 Released  May 2003.
@@ -804,7 +804,7 @@ Released  May 2003.
   Project may add 40 points for ranking in the alternatives
 </taglist></p>
 
-<sect> Version 3.5.9.0
+<sect id="3.5.9.0"> Version 3.5.9.0
 <p>
 
 Released  Mar 2003.
@@ -826,7 +826,7 @@ Released  Mar 2003.
   (rather than <file>/usr/share/doc/package/examples</file>).
 </taglist></p>
 
-<sect> Version 3.5.8.0
+<sect id="3.5.8.0"> Version 3.5.8.0
 <p>
 
 Released  Nov 2002.
@@ -850,7 +850,7 @@ Released  Nov 2002.
   are still a bug.
 </taglist></p>
 
-<sect> Version 3.5.7.0
+<sect id="3.5.7.0"> Version 3.5.7.0
 <p>
 
 Released  Aug 2002.
@@ -889,7 +889,7 @@ Added the last note in 3.5.6 upgrading checklist item regarding build
 rules, please see below
 </em></p>
 
-<sect> Version 3.5.6.0
+<sect id="3.5.6.0"> Version 3.5.6.0
 <p>
 
 Released  Jul 2001.
@@ -914,7 +914,7 @@ Released  Jul 2001.
   there is a new Build-Depend-Indep semantic.
 </taglist></p>
 
-<sect> Version 3.5.5.0
+<sect id="3.5.5.0"> Version 3.5.5.0
 <p>
 
 Released  May 2001.
@@ -968,7 +968,7 @@ Released  May 2001.
   </taglist></item>
 </taglist></p>
 
-<sect> Version 3.5.4.0
+<sect id="3.5.4.0"> Version 3.5.4.0
 <p> Released  Apr 2001.
 
 </p><p><taglist>
@@ -985,7 +985,7 @@ Released  May 2001.
 </item>
 </taglist></p>
 
-<sect> Version 3.5.3.0
+<sect id="3.5.3.0"> Version 3.5.3.0
 <p> Released Apr 2001
 
 </p><p><taglist>
@@ -1001,7 +1001,7 @@ Released  May 2001.
 </item>
 </taglist></p>
 
-<sect> Version 3.5.2.0
+<sect id="3.5.2.0"> Version 3.5.2.0
 
 <p> Released Feb 2001.
 
@@ -1013,7 +1013,7 @@ Released  May 2001.
 </item>
 </taglist></p>
 
-<sect> Version 3.5.1.0
+<sect id="3.5.1.0"> Version 3.5.1.0
 
 <p> Released Feb 2001.
 
@@ -1024,7 +1024,7 @@ Released  May 2001.
 </item>
 </taglist></p>
 
-<sect> Version 3.5.0.0
+<sect id="3.5.0.0"> Version 3.5.0.0
 
 <p> Released Jan 2001.
 
@@ -1035,7 +1035,7 @@ Released  May 2001.
 </item>
 </taglist></p>
 
-<sect> Version 3.2.1.1
+<sect id="3.2.1.1"> Version 3.2.1.1
 
 <p> Released Jan 2001.
 
@@ -1057,7 +1057,7 @@ Released  May 2001.
 </item>
 </taglist></p>
 
-<sect> Version 3.2.1.0
+<sect id="3.2.1.0"> Version 3.2.1.0
 
 <p> Released Aug 00.
 
@@ -1069,7 +1069,7 @@ Released  May 2001.
 </item>
 </taglist></p>
 
-<sect> Version 3.2.0.0
+<sect id="3.2.0.0"> Version 3.2.0.0
 
 <p> Released Aug 00.
 
@@ -1120,7 +1120,7 @@ Released  May 2001.
 </item>
 </taglist></p>
 
-<sect> Version 3.1.1.0
+<sect id="3.1.1.0"> Version 3.1.1.0
 
 <p> Released Nov 1999.
 
@@ -1131,7 +1131,7 @@ Released  May 2001.
 </item>
 </taglist></p>
 
-<sect> Version 3.1.0.0
+<sect id="3.1.0.0"> Version 3.1.0.0
 
 <p> Released Oct 1999.
 
@@ -1186,7 +1186,7 @@ Released  May 2001.
 </item>
 </taglist></p>
 
-<sect> Version 3.0.1.0
+<sect id="3.0.1.0"> Version 3.0.1.0
 
 <p> Released Jul 1999.
 
@@ -1198,7 +1198,7 @@ Released  May 2001.
 </item>
 </taglist></p>
 
-<sect> Version 3.0.0.0
+<sect id="3.0.0.0"> Version 3.0.0.0
 
 <p> Released Jun 1999.
 
@@ -1243,7 +1243,7 @@ Released  May 2001.
    Please note that section numbers below this point may not be up to date
 </em></p>
 
-<sect> Version 2.5.0.0
+<sect id="2.5.0.0"> Version 2.5.0.0
 
 <p> Released Oct 1998.
 
@@ -1282,7 +1282,7 @@ Policy Manual:
   </item>
 </list></p>
 
-<sect> Version 2.4.1.0
+<sect id="2.4.1.0"> Version 2.4.1.0
 
 <p> Released Apr 1998.
 </p>
@@ -1309,7 +1309,7 @@ Policy Manual:
 </item>
 </list></p>
 
-<sect> Version 2.4.0.0
+<sect id="2.4.0.0"> Version 2.4.0.0
 
 <p> Released Jan 1998
 
@@ -1385,7 +1385,7 @@ Policy Manual:
   </item>
 </taglist></p>
 
-<sect> Version 2.3.0.1, 2.3.0.0
+<sect id="2.3.0.0"> Version 2.3.0.1, 2.3.0.0
 
 <p> Released Sep 1997.
 
@@ -1402,7 +1402,7 @@ Policy Manual:
 <item>MUAs and MTAs have to use liblockfile</item>
 </list></p>
 
-<sect> Version 2.2.0.0
+<sect id="2.2.0.0"> Version 2.2.0.0
 
 <p> Released July 1997.
 
@@ -1429,7 +1429,7 @@ Policy Manual:
 <item>app-defaults files may not be conffiles</item>
 </list></p>
 
-<sect> Version 2.1.3.2, 2.1.3.1, 2.1.3.0
+<sect id="2.1.3.0"> Version 2.1.3.2, 2.1.3.1, 2.1.3.0
 
 <p> Released Mar 1997.
 
@@ -1447,7 +1447,7 @@ Policy Manual:
 </item>
 </list></p>
 
-<sect> Version 2.1.2.2, 2.1.2.1, 2.1.2.0
+<sect id="2.1.2.0"> Version 2.1.2.2, 2.1.2.1, 2.1.2.0
 
 <p> Released Nov 1996.
 
@@ -1455,7 +1455,7 @@ Policy Manual:
 <item>Some changes WRT shared libraries
 </list></p>
 
-<sect> Version 2.1.1.0
+<sect id="2.1.1.0"> Version 2.1.1.0
 
 <p> Released Sep 1996.
 
@@ -1468,7 +1468,7 @@ without consultation </item>
 <item>Shared libraries must be installed stripped </item>
 </list></p>
 
-<sect> Version 2.1.0.0
+<sect id="2.1.0.0"> Version 2.1.0.0
 
 <p> Released Aug 1996.
 
-- 
1.7.7.3

